docs/ERC-725.md

```diff
@@ -97,70 +97,81 @@ MUST be triggered when `execute` creates a new contract using the `operationType
 
 ERC165 identifier: `0x714df77c`
 
+Note: Using **web3.js** or **ether.js**, it is advised to call the overloaded ERC725Y functions using these notations:
+
+- The name with parameters: `myContract.methods['setData(bytes32,bytes)'](dataKey, dataValue)`
+- The signature: `myContract.methods['0x7f23690c'](dataKey, dataValue)`
+
 #### setData
 
 ```solidity
-function setData(bytes32 key, bytes memory value) public
+function setData(bytes32 dataKey, bytes memory dataValue) public
```

Comment:
I am not sure if we should keep the parameter names with or without underscore.

So `dataKey` -> `_dataKey` for instance

Reply:
@CJ42 The underscore is useful when the parameter name is the same as the storage variable name, in our case, it's a mapping named `store`. I don't see a reason why we should add an underscore.

Reply:
True, it is actually recommended as mixedCase in the Solidity Style Guide:
https://docs.soliditylang.org/en/v0.8.13/style-guide.html#function-argument-names